### PR TITLE
Add info about console.log

### PR DESCRIPTION
**Pull request by milep**
May need a better phrasing.

### CODE REVIEW

File: 5. Administrator Guides/Integrations/README.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ The class should have a method called **process_incoming_request**, this method
 
 The **process_incoming_request** method should return an object with a property **content** containing a valid Rocket.Chat message or an object with a property **error** that will be returned as the response of the request in JSON format and status code **400**.
 
+You can use the **console** to log and view the logs from Administration -> View Logs.
```
Comment (geekgonecrazy) [inline 5. Administrator Guides/Integrations/README.md]: Maybe we should include a link to: https://developer.mozilla.org/en-US/docs/Web/API/Console/log

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

**Comment by geekgonecrazy** [comment]
Thanks for contributing!  Left a note.  Might be useful to give some context as to what console is for those that don't understand :)

[edit] Made the adjustments.

File: 5. Administrator Guides/Integrations/README.md (hunk 1 above)
Comment (engelgabriel) [reply 5. Administrator Guides/Integrations/README.md]: @geekgonecrazy can you add your suggestion directly?

Comment (geekgonecrazy) [reply 5. Administrator Guides/Integrations/README.md]: @engelgabriel Ah looks like that is enabled now!  Sweet.  Changes made.

**Review by geekgonecrazy — DISMISSED** [review]

**Review by engelgabriel — APPROVED** [review]

**Pull request merged**